sdk/src/main/java/io/radar/sdk/RadarForegroundService.kt

```diff
@@ -56,8 +56,11 @@ class RadarForegroundService : Service() {
         val title = extras?.getString("title")
         val text = extras?.getString("text") ?: "Location tracking started"
         var icon = extras?.getInt("icon") ?: 0
-        icon = if (icon == 0) this.applicationInfo.icon else icon
-        val smallIcon = resources.getIdentifier(icon.toString(), "drawable", applicationContext.packageName)
+        var iconString = extras?.getString("iconString") ?: this.applicationInfo.icon.toString()
+        var smallIcon = resources.getIdentifier(iconString, "drawable", applicationContext.packageName) 
+        if (icon != 0){
+           smallIcon = resources.getIdentifier(icon.toString(), "drawable", applicationContext.packageName)  
+        }
```

Comment:
Did we test/QA whether or not if they previously had an icon showing (guess it'd just `resources.getIdentifier(icon.toString(), "drawable", applicationContext.packageName)`, that it still works?

Reply:
We need the fallback to `this.applicationInfo.icon` in line 59 of the original.